codalab/model/bundle_model.py

```diff
@@ -988,9 +988,11 @@ def save_bundle(self, bundle):
 
     def update_bundle(self, bundle, update, connection=None):
         """
-        Update a bundle's columns and metadata in the database and in memory.
-        The update is done as a diff: columns that do not appear in the update dict
+        Update a bundle's columns and metadata (default) or delete a bundle's metadata in the database and in memory.
```

Comment:
Improve the docstring: Say: for each key: value in `update`, add metadata pair (key, value), replacing any old value.  Note that metadata keys not in `update` are not affected.  Also, delete any metadata entries corresponding to keys `delete_keys`.  [See comment below]